generators/app/templates/build.xml

```diff
@@ -1,28 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project default="all" name="<%- title %>">
-    <description><%- desc %></description>
-    <xmlproperty file="expath-pkg.xml"/>
-    <property name="project.version" value="${package(version)}"/>
-    <property name="project.app" value="<%- title %>"/>
-    <property name="build.dir" value="build"/>
+  <description>
+    <null>
```

Comment:
@duncdrum I believe this unclosed element is the source of the error.

Reply:
Yup that’s it, no clue how that got in there.